sdk/python/v1beta1/kubeflow/katib/api/katib_client.py

```diff
@@ -280,6 +295,23 @@ def tune(
                 + exec_script
             )
 
+        resources = client.V1ResourceRequirements()
+        if isinstance(resources_per_trial, dict):
+            requests = {
+                "cpu": "200m",
+                "memory": "256Mi",
+            }
+            if "gpu" in resources_per_trial:
```

Comment:
Do you need to pop cpu and memory values from dict as well?

Reply:
Oh, I missed that!

Reply:
No worries! I wasn't familiar with the `update` method until @shipengcheng1230 suggested that we use it during our pair programming session yesterday! 😅